spring-cloud-stream-binder-kinesis/src/main/java/org/springframework/cloud/stream/binder/kinesis/properties/KinesisBinderConfigurationProperties.java

```diff
@@ -110,6 +131,8 @@ public Locks getLocks() {
 
 		private Integer timeToLive;
 
+		private Long intervalForKcl = null;
```

Comment:
I would call this just `interval` since we are going to support it in both Kinesis channel adapters implementations.